Task 1/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+
+        {
+            Console.ReadKey();
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("Николай Рубцов - Коза");
+
+            Console.ReadKey();
+            Console.ForegroundColor = ConsoleColor.Yellow;
+
+            Console.WriteLine("Побежала коза в огород.");
+            Console.ReadKey();
+            Console.WriteLine("Ей навстречу попался народ.");
+            Console.ReadKey();
+            Console.WriteLine("Как не стыдно тебе, егоза? —");
+            Console.ReadKey();
+            Console.WriteLine("И коза опустила глаза.");
+            Console.ReadKey();
+            Console.WriteLine("А когда разошелся народ,");
+            Console.ReadKey();
+            Console.WriteLine("Побежала опять в огород.");
+            Console.ReadKey();
+
+            Console.Clear();
+            Console.ForegroundColor = ConsoleColor.Gray;
+            Console.WriteLine("Николай Рубцов - Коза");
+            Console.WriteLine("Побежала коза в огород.");
+            Console.WriteLine("Ей навстречу попался народ.");
+            Console.WriteLine("Как не стыдно тебе, егоза? —");
+            Console.WriteLine("И коза опустила глаза.");
+            Console.WriteLine("А когда разошелся народ,");
+            Console.WriteLine("Побежала опять в огород.");
+
+            ConsoleKeyInfo consoleKeyInfo = Console.ReadKey();
```

Comment:
Зачем объявлять переменную, если она потом не используется
```suggestion
            Console.ReadKey();
```